include/types.h

```diff
@@ -4,6 +4,7 @@
 #include <functional>
 #include <utility>
 
+
```

Comment:
Do we usually have 2 newlines here? (nitpick)

Reply:
Now we don't.